README.md

```diff
@@ -7,7 +7,7 @@ This module provisions by default a set of basic AWS Config Rules. In order to a
 
 If you would like to authorize other accounts to aggregate AWS Config data, the account IDs and regions can also be passed via the variable `aws_config` using the attributes `aggregator_account_ids` and `aggregator_regions` respectively. 
 
-NOTE: The `audit` account will be automatically authorized to aggregate AWS Config data from the other 2 core acccounts in the following regions: `eu-central-1` and `eu-west-1`.
+NOTE: AWS Control Tower automatically authorizes the `audit` account to aggregate AWS Config data for the whole Organization, so there is no need to specified it here. The `master` account is the only exception but this module contains resources that fix that.
```

Comment:
```suggestion
NOTE: AWS Control Tower automatically authorizes the `audit` account to aggregate AWS Config data for all the accounts in the organization, so there is no need to specify any of these accounts. The `master` account is the only exception but this module contains resources to address this. 
```

Reply:
Actually, what I meant is that you don't to specify the `audit` account in the list because it will already have the required permissions and not the other accounts...is it confusing?

Reply:
Ah yes you are right, that's not how I read this! The last sentences confuses me, can we be more specific here? Which sort of accounts do you need to specify here? All non-core accounts?

Reply:
I've updated the note...can you take a look again?